doc/install/containers.rst

```diff
@@ -3,6 +3,20 @@
 Ceph Container Images
 =====================
```

Comment:
we somehow have to make cephadm users aware of it, as they typically will not look into install/containers. 

What about simply moving this file to doc/cephadm?

Reply:
@sebastian-philipp Is it your vote that we move all of the content in containers.rst over to doc/cephadm?

Reply:
wdyt? Either we link it from cephadm, or we move it there.

Reply:
I've been sitting here chewing my lip and thinking about it, and I think I'm just going to add a link on the cephadm page. If after a few days we don't think that that's sufficient, I'll move the matter in this page to the cephadm page. (The cephadm page has bigger problems at the moment, in my opinion, anyway.)

Reply:
2 cents I wasn't pinged for, I agree with @zdover23 - a cross ref should be fine in this instance. A short sentence and "here's where to go". If there's complaints later, that can be rectified.